discovery-provider/src/tasks/index.py

```diff
@@ -189,7 +189,7 @@ def save_and_get_skip_tx_hash(session, redis):
         return indexing_error['txhash']
     return None
 
-def index_blocks(self, db, blocks_list):
+def index_blocks(self, db, event_bus, blocks_list):
```

Comment:
since event_bus is part of the update_task, do you even need this modification? it can be picked off from `update_task` like `redis` and `web3` etc

Reply:
for ex., we use the task ref to get abi_values here https://github.com/AudiusProject/audius-protocol/blob/4cda548cb845a21d9ab1dae54818d273af152482/discovery-provider/src/tasks/social_features.py#L25

Reply:
will change